tests/functional/spec/previous.spec.js

```diff
@@ -0,0 +1,50 @@
+const helpers = require('../helpers');
+const EmploymentStatusBlockPage = require('../generated_pages/hub_and_spoke/employment-status.page.js');
+const HubPage = require('../base_pages/hub.page.js');
+const schema = 'test_hub_complete_sections.json';
+
+describe('Choose another section link', function () {
+
+  it('When a user gets to initial question, then the previous location link should not be displayed', function () {
+    return helpers.openQuestionnaire(schema)
+      .then(() => {
+        return browser
+          .getSource().should.not.eventually.contain('Previous link click');
```

Comment:
The previous link is available so we shouldn't need to do `getSource`. Something like 
```js
.isExisting(EmploymentStatusBlockPage.previous()).should.eventually.be.false;
```

Same comment for the other occurrences.

Reply:
Thanks, changed all tests to utilise .previous()